tools/version.txt

```diff
@@ -0,0 +1 @@
+1.15.1-dev
```

Comment:
Setuptools will handle the task for us, so we can continue using the legacy format.

Reply:
This is something setuptools does because it's nice to do; not because it has to. I think changing to PyPA compliant is a small change that would reduce the risk of something breaking in the future. (Honestly I don't think many people would even notice beyond some of the core devs.)

Reply:
PyPA [does allow `-dev`](https://packaging.python.org/en/latest/specifications/version-specifiers/#development-release-separators) but probably better to use a dot since it's perferred

Reply:
Thanks. TIL.